astropy/io/ascii/tests/test_rst.py

```diff
@@ -3,13 +3,12 @@
 from io import StringIO
 
 import numpy as np
+from numpy.testing import assert_allclose, assert_equal
```

Comment:
7 occurrences left to clean up here
```suggestion
from numpy.testing import assert_allclose
```